src/Cli/ConfigGenerator.cs

```diff
@@ -222,9 +222,9 @@ public static bool TryCreateRuntimeConfig(InitOptions options, FileSystemRuntime
 
             if (runtimeBaseRoute is not null)
             {
-                if (!Enum.TryParse(options.AuthenticationProvider, ignoreCase: true, out EasyAuthType authMode) || authMode is not EasyAuthType.StaticWebApps)
+                if (!Enum.TryParse(options.AuthenticationProvider, ignoreCase: true, out EasyAuthType authMode) || authMode is not EasyAuthType.StaticWebApps or EasyAuthType.EasyAuth or EasyAuthType.None)
```

Comment:
Lets remove the change about EasyAuth.

Reply:
I've implemented requested changes. I'm resolving this comment chain.